playwright.config.js

```diff
@@ -43,28 +49,42 @@ const config = {
 
   /* Configure projects for major browsers */
   projects: [
+    {
+      name: "setup",
+      testMatch: "**/*.setup.mjs",
+    },
     {
       name: "chromium",
+      testDir: "./tests",
       use: {
         ...devices["Desktop Chrome"],
+        launchOptions: {
+          slowMo: 50,
```

Comment:
This should not be included by default. It should only be used for headed testing.

Reply:
removed done